build.gradle.kts

```diff
@@ -1,8 +1,33 @@
 plugins {
   id("otel.java-conventions")
+  id("java-library")
+  id("maven-publish")
 }
 
 repositories {
   mavenCentral()
   mavenLocal()
 }
+
+dependencies {
+  implementation(project(":proto"))
+}
+
+group = "io.opentelemetry"
+version = "1.0.0"
```

Comment:
we shouldn't use 1.0.0 here, but rather the release version. I'm not sure what's the best way to achieve this, perhaps we could add do something like [this](https://github.com/open-telemetry/opentelemetry-java/blob/main/version.gradle.kts)